mPower2/mPower2/View Controllers/StudyBurstViewController.swift

```diff
@@ -173,12 +170,61 @@ class StudyBurstViewController: UIViewController {
         }
     }
     
-    func showCompletionView() {
+    func showCompletionTask() {
         // TODO: syoung 06/20/2018 Update title/text to "congradulations" with party sprinkles.
         self.navFooterView.nextButton?.setTitle(Localization.buttonDone(), for: .normal)
         
-        // Call through to the today view to show the completion tasks.
-        (self.navigationController?.viewControllers.first as? TodayViewController)?.showStudyBurstCompletionView()
+        // If there is a task to do today, then push it.
+        if let taskPath = studyBurstManager.completionTaskPath() {
+            let vc = RSDTaskViewController(taskPath: taskPath)
+            vc.delegate = self
+            self.present(vc, animated: true, completion: nil)
+        }
```

Comment:
WARNING: Code Stomp. Please revert. 

See comment below for explanation or ping me on Slack if you still have questions. It's super-confusing. Sorry.